src/js/disability-benefits/686/config/form.js

```diff
@@ -1,42 +1,140 @@
 // import fullSchema from 'vets-json-schema/dist/686-schema.json';
+import _ from 'lodash';
 
 import IntroductionPage from '../containers/IntroductionPage';
 import ConfirmationPage from '../containers/ConfirmationPage';
+import fullSchema686 from 'vets-json-schema/dist/21-686C-schema.json';
+import fullNameUI from '../../../common/schemaform/definitions/fullName';
+import ssnUI from '../../../common/schemaform/definitions/ssn';
+import * as address from '../../../common/schemaform/definitions/address';
 
-// const { } = fullSchema.properties;
+import ssnOrVaUI from '../../../common/schemaform/definitions/ssnOrVafile';
+import { relationshipLabels } from '../helpers';
 
-// const { } = fullSchema.definitions;
+const { claimantEmail } = fullSchema686.properties;
+
+const { fullName, ssn } = fullSchema686.definitions;
 
 const formConfig = {
   urlPrefix: '/',
   submitUrl: '/v0/api',
-  trackingPrefix: '686',
+  trackingPrefix: '686-',
   introduction: IntroductionPage,
   confirmation: ConfirmationPage,
-  formId: '686',
+  formId: '21-686C',
   version: 0,
-  prefillEnabled: true,
+  prefillEnabled: false,
   savedFormMessages: {
     notFound: 'Please start over to apply for declaration of status of dependents.',
     noAuth: 'Please sign in again to continue your application for declaration of status of dependents.'
   },
   title: 'Declaration of status of dependents',
   defaultDefinitions: {
+    fullName,
+    ssn
   },
   chapters: {
-    chapter1: {
-      title: 'Chapter 1',
+    veteranInformation: {
+      title: 'Veteran Information',
       pages: {
-        page1: {
-          path: 'first-page',
-          title: 'First Page',
+        veteranInformation: {
+          path: 'veteran-information',
+          title: 'Veteran Information',
           uiSchema: {
+            fullName: fullNameUI,
+            ssnOrVa: _.merge({}, ssnOrVaUI, {
+              'ui:title': 'Social Security number or VA file number',
+              'ui:errorMessages': {
+                pattern: 'Your VA file number must be between 7 to 9 digits'
+              }
+            }),
+            relationship: {
+              'ui:title': 'Relationship to Veteran',
+              'ui:widget': 'radio',
+              'ui:options': {
+                labels: relationshipLabels
+              }
+            },
+            'view:applicantInfo': {
+              fullName: {
+                'ui:title': 'Applicant Information',
+                first: {
```

Comment:
You can do something similar here to what you did with `ssnOrVa`, and use `_.merge(fullNameUI, ...)`. That will let you add the `ui:required` fields without having to redefined all the titles and options.